### DISCUSSION TITLE
Is there a way to remove background blur?

### DISCUSSION THREAD

**Discussion by aharoJ** [asmvik/yabai#1832] [General]
Hey, I was wondering if there is a way to remove the background blur once we active the Limelight? I like how it highlights my focus window but I dont want my terminals to have a background blur by default. Anyone know if there is a way to disable background blur?

**Comment by asmvik** [comment]
https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc

cmd+f blur
